content/it/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,160 @@
+---
+reviewers:
+- bgrant0607
+- mikedanese
```

Comment:
Please remove the 

```bash
reviewers:
- bgrant0607
- mikedanese
```

part from this file

Reply:
Hi :)
I think i removed everything that you ask
Thanks

Reply:
@micheleberardi thanks. you updated this file correctly. However, can you please remove the following files as well:
```bash
content/it/case-studies/OWNERS
content/it/blog/OWNERS
```